cf_xarray/tests/test_accessor.py

```diff
@@ -295,11 +295,9 @@ def test_dataarray_plot(obj):
     plt.close()
 
     rv = obj.cf.plot(x="X", y="Y", col="T")
```

Comment:
Now does this work. that will test the changes in this PR

```suggestion
    rv.map_dataarray(xr.plot.contourf, rv, x="X", y="Y", col="T")
```